### PR TITLE
feat(Spanner): DML returning samples

### PR DESCRIPTION
**Pull request by yash30201**
Samples showcasing how to use dml returning feature in spanner using php client library.

### CODE REVIEW

File: spanner/src/delete_dml_returning.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        . 'THEN RETURN *',
+        [
+            'parameters' => [
+              'firstName' => 'Melissa',
+            ]
+        ]
+    );
+    foreach ($result->rows() as $row) {
+        printf(
+            'Row (%s, %s, %s) deleted\n' . PHP_EOL,
```
Comment (saranshdhingra) [inline spanner/src/delete_dml_returning.php:55]:
```suggestion
            'Row (%s, %s, %s) deleted' . PHP_EOL,
```

Can you remove it from other samples too?

File: spanner/src/delete_dml_returning.php [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/spanner/README.md
+ */
+
+namespace Google\Cloud\Samples\Spanner;
+
+// [START spanner_delete_dml_returning]
+use Google\Cloud\Spanner\SpannerClient;
+
+/**
+ * Delete data from the given database using DML returning feature.
```
Comment (saranshdhingra) [inline spanner/src/delete_dml_returning.php:30]:
```suggestion
 * Delete data from the given database using DML returning.
```

nit: Can you please remove the word 'feature' from other samples too?

File: spanner/test/spannerPgTest.php [hunk 3]
```diff
@@ -363,6 +363,37 @@ public function testDmlGettingStartedUpdate()
         $this->assertStringContainsString('Marketing budget updated.', $output);
     }
 
+    /**
+     * @depends testCreateDatabase
+     */
+    public function testDmlReturningInsert()
+    {
+        $output = $this->runFunctionSnippet('pg_insert_dml_returning');
+        $this->assertStringContainsString('Melissa', $output);
```
Comment (saranshdhingra) [inline spanner/test/spannerPgTest.php:372]: These assertions should check for the whole line that was used in `printf`.

**Review by saranshdhingra — CHANGES_REQUESTED** [review]

File: spanner/src/delete_dml_returning.php (hunk 1 above)
Comment (yash30201) [reply spanner/src/delete_dml_returning.php]: Thanks, fixed in [de8e44b](https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1721/commits/de8e44bd199f023ac080a6a21e409e6d98943f58).

File: spanner/test/spannerPgTest.php (hunk 3 above)
Comment (yash30201) [reply spanner/test/spannerPgTest.php]: Thanks, fixed in [de8e44b](https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1721/commits/de8e44bd199f023ac080a6a21e409e6d98943f58)

File: spanner/src/delete_dml_returning.php (hunk 2 above)
Comment (yash30201) [reply spanner/src/delete_dml_returning.php]: Thanks, fixed in [de8e44b](https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1721/commits/de8e44bd199f023ac080a6a21e409e6d98943f58)

File: spanner/src/update_dml_returning.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+{
+    $spanner = new SpannerClient();
+    $instance = $spanner->instance($instanceId);
+    $database = $instance->database($databaseId);
+
+    $transaction = $database->transaction();
+
+    // DML returning sql update query
+    $result = $transaction->execute(
+        'UPDATE Singers SET LastName = @lastName WHERE SingerId = @singerId THEN RETURN *',
```
Comment (saranshdhingra) [inline spanner/src/update_dml_returning.php:45]: nit: Split this.

Comment (yash30201) [reply spanner/src/update_dml_returning.php]: Fixed

**Review by saranshdhingra — APPROVED** [review]

**Pull request merged**